src/dexScreener.ts

```diff
@@ -0,0 +1,95 @@
+export async function uniV2PairValue(
```

Comment:
TODO(bh2smith): Build a DexScreener API SDK - analogous to [Zerion SDK](https://github.com/bh2smith/zerion-sdk/)